lib/ssz_ex.ex

```diff
@@ -331,39 +331,25 @@ defmodule LambdaEthereumConsensus.SszEx do
     root |> hash_nodes(serialized_len)
   end
 
+  # Currently, we are not using this one.
   def merkleize_chunks(chunks, leaf_count \\ nil) do
     chunks_len = chunks |> get_chunks_len()
 
     if chunks_len == 1 and leaf_count == nil do
       chunks
     else
-      node_count = 2 * leaf_count - 1
-      interior_count = node_count - leaf_count
-      leaf_start = interior_count * @bytes_per_chunk
-      padded_chunks = chunks |> convert_to_next_pow_of_two(leaf_count)
-      buffer = <<0::size(leaf_start * @bits_per_byte), padded_chunks::bitstring>>
-
-      new_buffer =
-        1..node_count
-        |> Enum.filter(fn x -> rem(x, 2) == 0 end)
+      power = leaf_count |> compute_pow()
+      height = power + 1
+      first_layer = chunks |> convert_to_next_pow_of_two(leaf_count)
+
+      final_layer =
+        1..(height - 1)
         |> Enum.reverse()
```

Comment:
Since this is a range, you can just invert the range:
```suggestion
        (height - 1)..1
```